src/Http/Http.Abstractions/src/Routing/RouteValueDictionary.cs

```diff
@@ -2,35 +2,54 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
 using System.Collections;
-using System.Collections.Concurrent;
 using System.Diagnostics;
 using System.Diagnostics.CodeAnalysis;
 using System.Linq;
-using System.Reflection.Metadata;
 using System.Runtime.CompilerServices;
+#if !COMPONENTS
+using System.Collections.Concurrent;
+using System.Reflection.Metadata;
 using Microsoft.AspNetCore.Http.Abstractions;
-using Microsoft.AspNetCore.Routing;
 using Microsoft.Extensions.Internal;
+using Microsoft.AspNetCore.Routing;
+#endif
 
+#if !COMPONENTS
 [assembly: MetadataUpdateHandler(typeof(RouteValueDictionary.MetadataUpdateHandler))]
+#endif
 
+#if COMPONENTS
+namespace Microsoft.AspNetCore.Components.Routing;
```

Comment:
It's a public type in ASP.NET, so it causes issues in Blazor Server and SSR. There are a couple of other instances where I had to do something similar because of `InternalsVisibleTo` or because the type was public in ASP.NET.